src/index.ts

```diff
@@ -21,7 +21,7 @@ export default new Resolver({
 		const result = attemptResolve(filePath, pathsMap, logger);
 		logger.verbose({ message: `Result: ${result}` });
 		return {
-			filePath: result,
+			filePath: path.join(__dirname, '../../../../', result),
```

Comment:
Yep. Not the best solution, but I haven't found a different way to get the project root or project package.json location to resolve the absolute path.

Reply:
I got this error:

```
parcel-resolver-tspaths: The "path" argument must be of type string. Received null
  at Object.resolve (C:\Users\Allen\Desktop\Personal\tauri-parcel-react\node_modules\parcel-resolver-tspaths\dist\min\index.min.js:1:2965)
```

my config:
```
// tsconfig:
"baseUrl": ".",
"paths": {
    "@/*": ["client/src/*"],
    "~/*": ["./*"]
 },

// load:
import Layout from "@/components/Layout";
```

Reply:
maybe u can use something like this:

```ts
const appDirectory = fs.realpathSync(process.cwd());

const resolveApp = (...relativePaths) => path.resolve(appDirectory, ...relativePaths);
```

Reply:
Seems like parcel changed the requirements and started to enforce it. Fixed in the coming version